### DISCUSSION TITLE
Detect H1 to H6 tags

### DISCUSSION THREAD

**Discussion by leopadm** [getgauge/taiko#2390] [Ideas]
Hello, I'm currently using taiko with gauge to automatically test some websites the company is developing.
I encountered some SEO websites proposing to detect H1 tags just by entering the domain of the website. Thus, my question is: would it be possible with taiko to detect tags, such as H1?

Thank you 😄 
Mimopoldo

**Comment by sriv** [comment]
HI, yes you should be able to do this using the [`$`](https://docs.taiko.dev/api/$/) selector which lets you use CSS/XPath
